src/current/cockroachcloud/resource-usage.md

```diff
@@ -39,6 +41,88 @@ To understand these resources, you need to understand a bit about the CockroachD
+
+<section class="filter-content" markdown="1" data-scope="standard">
+
+Substantial RU consumption (greater than 100 RU/second) is usually caused by SQL queries issued by the application. This can be confirmed by verifying that RU consumption tightly follows changes to the application’s SQL QPS (queries per second). On the CockroachDB {{ site.data.products.cloud }} Console [**Overview** metrics page]({% link cockroachcloud/metrics-overview.md %}), you can compare the [**Request Units** chart]({% link cockroachcloud/metrics-overview.md %}#request-units) to the [**SQL Statements** chart]({% link cockroachcloud/metrics-overview.md %}#sql-statements). Assuming the charts correlate, then reducing Request Unit consumption is about optimizing application SQL queries ([`SELECT`]({% link {{site.current_cloud_version}}/select-clause.md %}), [`UPDATE`]({% link {{site.current_cloud_version}}/update.md %}), [`INSERT`]({% link {{site.current_cloud_version}}/insert.md %}), [`DELETE`]({% link {{site.current_cloud_version}}/delete.md %})).
+
+In the CockroachDB {{ site.data.products.cloud }} Console, you can monitor your cluster's SQL activity on the [**Statements**]({% link cockroachcloud/statements-page.md %}) and [**Transactions**]({% link cockroachcloud/transactions-page.md %}) pages. You can sort queries by the time they took to process, the number of rows processed, or the number of bytes read to see which queries are using the most resources. If you have queries that return more data than needed or have long runtimes, those are good candidates for optimization. 
+
+### Expensive queries
+
+Expensive queries, especially `FULL SCAN` operations, are the most common cause of unexpected RU consumption increases and a good place to begin investigating your consumption. To diagnose expensive queries:
+
+1. Navigate to the [**Statements** tab]({% link cockroachcloud/statements-page.md %}) of your cluster's **SQL Activity** page in the {{ site.data.products.cloud }} Console.
+1. Click on the title of the **Rows Processed** column to sort your queries by the number of rows processed. 
+ 
+    For most queries, total rows processed should be no more than a few hundred. Read queries are often more expensive than write queries.
+  
+1. Next, sort the queries by the **Bytes Read** column. Most queries should read no more than a few kilobytes per row.
+
+If any queries are more expensive than expected, you can use the [`EXPLAIN ANALYZE` SQL command](https://www.cockroachlabs.com/docs/{{site.current_cloud_version}}/explain-analyze) for an estimate of the RUs consumed. Efficient queries generally consume fewer RUs, so the guidelines for [Optimizing Query Performance](https://www.cockroachlabs.com/docs/stable/performance-best-practices-overview) can be applied here. You can also refer to [Cockroach Labs Blog - How to troubleshoot and optimize query performance in CockroachDB](https://www.cockroachlabs.com/blog/query-performance-optimization/) for further information.
```

Comment:
recommend adding an anchor link to the EXPLAIN ANALYZE section that describes the RU estimate in the output

https://www.cockroachlabs.com/docs/stable/explain-analyze#global-properties

Reply:
updated